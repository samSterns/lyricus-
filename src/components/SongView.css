.Lyrics {
    margin: 5%;
    margin-right: 20%;
    padding: 5%;
    height: auto;
    width: auto;
    font-size: 100%;
    line-height: 22px;
    font: Roboto;
    border: 1px solid black;
    border-radius: 5px;
    box-shadow: 12px 12px 2px 1px #F2F2F2;
    background-color: #A53F2B;
    opacity: .8;
}

p {
    position: relative;
    font-size: 100%;
    line-height: 24px;
    text-align: center;
    font: Roboto;
    color: black;
    top: 4.7em;
    opacity: .9;
    color: #F2F2F2;
    padding: 5%;
}
