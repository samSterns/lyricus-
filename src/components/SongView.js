import React from 'react';
import useSong from '../hooks/useSong.js';


const SongView = () => {
  const { song } = useSong();

  return (
    <p>{song}</p>

  );
};

export default SongView;
