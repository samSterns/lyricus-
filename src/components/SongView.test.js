import React from 'react';
import { shallow } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';
import SongView from './SongView';

describe('SongView component', () => {
  it('renders an Song View', () => {
    const wrapper = shallow(<MemoryRouter initialEntries={[{ pathname: '/', key: 'testKey' }]}><SongView /></MemoryRouter>);
    expect(wrapper).toMatchSnapshot();
  });
});
