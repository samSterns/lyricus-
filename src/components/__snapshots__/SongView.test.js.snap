// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SongView component renders an Song View 1`] = `
<Router
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "canGo": [Function],
      "createHref": [Function],
      "entries": Array [
        Object {
          "hash": "",
          "key": "testKey",
          "pathname": "/",
          "search": "",
        },
      ],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "index": 0,
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "key": "testKey",
        "pathname": "/",
        "search": "",
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <SongView />
</Router>
`;
